src/style.css

```diff
@@ -1 +1,57 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0,05);
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__li {
+  display: inline-block;
+  font-family: "Roboto", sans-serif;
```

Comment:
Maybe tests haven't passed cause of this 'np' thing in HTML, try to move it to HTML again